### PR TITLE
Customer Tags API Documentation v1.0

### PR DESCRIPTION
**Pull request by siddtewari**

### CODE REVIEW

File: customer_tags_api.md [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+Content-Type: application/json
+</code></pre> 
+
+###Request Body Example for Tagging AWS Accounts:
+<pre><code>{
+  "assets" :
+    [
+      {
+        "type" : "AwsAccount",
+        "owner_id" : "1467",
```
Comment (vpillai) [inline customer_tags_api.md]: This is always a 12 digit identifier.

File: customer_tags_api.md [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+        }
+      }
+    ]
+}
+</code></pre>
+
+Where:
+
+* `aws_account_id` is the CloudHealth AWS Account ID (not your customer account) that owns the instance,  
+* `owner_id` is the AWS instance ID to be tagged, and
```
Comment (vpillai) [inline customer_tags_api.md]: owner_id is the Amazon assigned ID for the AWS Account.

File: customer_tags_api.md [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+* `tags` is a hash (object) of key value pairs
+
+###Request Body Example for Tagging AWS Assets:
+<pre><code>{
+  "assets":
+    [
+      {
+        "aws_account_id": 90,
```
Comment (vpillai) [inline customer_tags_api.md]: NB: This is the primary key of the AwsAccount.

File: customer_tags_api.md (hunk 1 above)
Comment (siddtewari) [inline customer_tags_api.md]: done... changes coming in next PR

File: customer_tags_api.md (hunk 2 above)
Comment (siddtewari) [inline customer_tags_api.md]: noted as such

File: customer_tags_api.md (hunk 3 above)
Comment (siddtewari) [inline customer_tags_api.md]: noted!

File: customer_tags_api.md [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+        }
+      },
+      ...
+    ]
+}</code></pre>  
+  
+
+Where:  
+
+* `aws_account_id` is the CloudHealth Account ID for the AWS account that owns the instance to be tagged. You can find the `aws_account_id` by clicking on the bulls eyes icon from the [AWS account page](https://apps.cloudhealthtech.com/aws_accounts/) and grabbing the number at the end of the generated URL
```
Comment (vpillai) [inline customer_tags_api.md]: bulls-eye

**Comment by vpillai** [comment]
:+1:

File: customer_tags_api.md (hunk 4 above)
Comment (siddtewari) [inline customer_tags_api.md]: A bull technically has two eyes but sadly our icon has only one eye.

**Pull request merged**